spec/tasks/role_spec.rb

```diff
@@ -51,6 +51,15 @@
         allow(resource).to receive(:role).with(ElasticWhenever::Task::Role::NAME).and_return(nil)
       end
 
+      it "returns false" do
```

Comment:
I think this test case is incorrect. Please remove this one :pray:

Reply:
done.